modular_skyrat/modules/hydra/body_markings.dm

```diff
@@ -0,0 +1,28 @@
+//Manages Hydra heads.
```

Comment:
why this

Reply:
Showing what the files for incase it gets expanded into a spaghetti nightmare

Reply:
no like, why is it all commented out